app/controllers/notes_controller.rb

```diff
@@ -28,8 +39,9 @@ def create
     if note.nil?
       note = Note.new(video_url: video_url, user: current_user)
       transcript = TranscriptGenerator.new(video_url).call
+      note.transcript = transcript
       memo = NoteGenerator.new(transcript).call
-      note.memo = transform_bracketed_text(memo)
+      note.memo = memo
```

Comment:
instead of storing the transformed memo (with Markdown converted to HTML, and bracketed text transformed), store the original memo, then transform it when we hit the show page.